txs/rebond_tx.go

```diff
@@ -0,0 +1,55 @@
+package txs
```

Comment:
Are these bond_tx and rebond_tx actually reimplementing validator changes in preparation for validator_tx or are they just getting back and mocked?

Reply:
cool